src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -69,14 +152,15 @@ void setUserStore(UserStore userStore) {
 	@Override
 	public void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException 
 	{
-	    List<Conversation> conversations = conversationStore.getAllConversations();
-	    List<User> users = userStore.getAllUsers();
-	    List<Message> messages = messageStore.getAllMessages();
-	    request.setAttribute("users",users);
-	    request.setAttribute("conversations", conversations);
-	    request.setAttribute("messages", messages);
+		List<User> users = userStore.getAllUsers();
+		List<Conversation> conversations = conversationStore.getAllConversations();
+	    List<Message> messages = MessageStore.getAllMessages();
+	    HashMap<Instant,HashMap<UUID,String>> instantByInstance=buildHashMap(conversations,users,messages);

```

Comment:
What does the variable name instantByInstance mean? I could be missing something, but it kinda confused me.